rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -119,6 +118,8 @@ class Publisher : public PublisherBase
     }
   }
 
+  [[deprecated("publishing an unique_ptr is prefered when using intra process communication."
+               " If using a shared_ptr, use publish(*msg).")]]
```

Comment:
Do not wrap like this, instead indent only two spaces, if you want the strings aligned then line break after the `(`.